test/unit/BondingManager.js

```diff
@@ -1799,26 +1800,45 @@ contract("BondingManager", accounts => {
             })
         })
 
-        describe("transcoder is not active", () => {
-            it("still decreases transcoder's bondedAmount", () => {
-                it("still decreases transcoder's bondedAmount", async () => {
-                    const startBondedAmount = (await bondingManager.getDelegator(transcoder))[0]
-                    await fixture.ticketBroker.execute(
-                        bondingManager.address,
-                        functionEncodedABI(
-                            "slashTranscoder(address,address,uint256,uint256)",
-                            ["address", "uint256", "uint256", "uint256"],
-                            [transcoder, constants.NULL_ADDRESS, PERC_DIVISOR / 2, 0]
-                        )
+        describe("transcoder is not active but registered", () => {
```

Comment:
I think this should be "transcoder is in not active but is in pool".

Reply:
fixed in e6d92cf